src/Domain.LinnApps/Reports/PurchaseOrdersReportService.cs

```diff
@@ -183,15 +193,16 @@ private void AddSupplierReportColumns(SimpleGridLayout reportLayout)
                         new AxisDetailsModel("QtyOrd", "Qty Ordered", GridDisplayType.TextValue),
                         new AxisDetailsModel("QtyRec", "Qty Rec", GridDisplayType.TextValue),
                         new AxisDetailsModel("QtyInv", "Qty Inv", GridDisplayType.TextValue),
-                        new AxisDetailsModel("NetTotal", "Net Total", GridDisplayType.TextValue),
+                        new AxisDetailsModel("BaseNetTotal", "Net Total (GBP)", GridDisplayType.Value),
+                        new AxisDetailsModel("Currency", "Currency", GridDisplayType.TextValue),
+                        new AxisDetailsModel("NetTotalCurrency", "Net Total (Currency)", GridDisplayType.Value),
```

Comment:
I haven't used this for a while but pretty sure you can specify the decimal places in here?

Reply:
I must not have actually checked the AxisDetailsModel class yesterday - you can't via the constructor, but you're right it's on the class. Have just defined the decimal columns at the top and set the decimalPlaces before adding them to the columns list